docs/usage/screening.md

```diff
@@ -106,7 +113,11 @@ If there are several designs to be run, you can input a folder containing all th
 
 This label is not mandatory as in case you are running time series. If you wish to run MAGeCK MLE with the day0 label you can do so by specifying `--day0_label` and the sample names that should be used as day0. The contrast will then be automatically adjusted for the other days.
 
-### MAGECKFlute
+#### With the contrast file
+
+To run MAGeCK MLE, you can define the contrasts as previously stated in the last section with --contrasts your_file.txt and also specify `--mle`.
```

Comment:
```suggestion
To run MAGeCK MLE, you can define the contrasts as previously stated in the last section with `--contrasts <your_file.txt>` and also specify `--mle`.
```

I would put all occurences of variable file names ("your_file.txt" in this case) within < > throughout the docs. Not a dealbreaker though.